modules/bastion/README.md

```diff
@@ -0,0 +1,121 @@
+# OpenTelekomCloud Bastion Terraform module
+
+![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/opentelekomcloud/terraform-opentelekomcloud-modules)
+![Build (latest by date)](https://zuul.otc-service.com/api/tenant/eco/badge?project=opentelekomcloud/terraform-opentelekomcloud-modules&pipeline=check&branch=main)
+
+_This module aims to create a module to create SSH Bastion Host OpenTelekomCloud provider._
+
+## Where to find module documentations
+
+You can find different documentations versioned by terraform registry [here](https://registry.terraform.io/modules/opentelekomcloud/modules/opentelekomcloud/latest).
+
+## Terraform versions
+
+Terraform 0.13 or higher.
+
+## Usage
+
+```hcl
+resource "opentelekomcloud_vpc_v1" "vpc_1" {
```

Comment:
would be nice to use here new vpc module

Reply:
vpc module is now used in examples instead of resource